src/pybind/mgr/orchestrator.py

```diff
@@ -948,17 +962,22 @@ class InventoryNode(object):
     When fetching inventory, all Devices are groups inside of an
     InventoryNode.
     """
-    def __init__(self, name, devices=None):
-        # type: (str, inventory.Devices) -> None
+    def __init__(self, name, devices=None, labels=None):
```

Comment:
Does this really belong into InventoryNode?

Reply:
I don't care.  I added it there because that's what get_hosts() returns, and that's what is used for 'orchestrator host ls'

Reply:
For Rook, I really need to return a lot more information. https://trello.com/c/nkhztNpC/121-enhance-host-ls